sourcecode/npm/components/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM node:16.13
+FROM node:16-alpine
 WORKDIR /app
+RUN apk add curl
```

Comment:
This should be run with `--no-cache` to keep the size of the Docker image down.

Reply:
Updated